client/src/components/sologame/SegmentedBar.vue

```diff
@@ -147,6 +139,8 @@ export default class SegmentedBar extends Vue {
   }
 
   mounted() {
+    // FIXME: consider if we really want to use scroll wheel to increment / decrement the value, keyboard input might be
+    // better for accessibility (+ / -) (<- ->)?
```

Comment:
keyboard controls are a good idea, L/R arrow keys and enter for investing/continuing through events

the scroll thing only happens when the cursor is over the input bar so it hopefully wouldn't interfere with anything too much, though probably unnecessary if we have keyboard controls